CHANGELOG.md

```diff
@@ -6,10 +6,10 @@
 
 ## 6.18.1
 
-- fix(ember): use _backburner if it exists (#4603)
-- feat(gatsby): Upgrade Sentry Webpack Plugin to 1.18.8 (#4636)
-- feat(nextjs): Upgrade Sentry Webpack Plugin to 1.18.8 (#4643)
-- fix(nextjs): webpack as optional peer-dependency (#4634)
+- fix(ember): use _backburner if it exists ([#4603](https://github.com/getsentry/sentry-javascript/pull/4603))
+- feat(gatsby): Upgrade Sentry Webpack Plugin to 1.18.8 ([#4636](https://github.com/getsentry/sentry-javascript/pull/4636))
+- feat(nextjs): Upgrade Sentry Webpack Plugin to 1.18.8 ([#4643](https://github.com/getsentry/sentry-javascript/pull/4643))
+- fix(nextjs): webpack as optional peer-dependency ([#4634](https://github.com/getsentry/sentry-javascript/pull/4634))
```

Comment:
Just a question: Do we need these URL links here? Asking because some release notes do not have them (https://github.com/getsentry/sentry-javascript/releases/tag/6.17.6) while (most) others do.

Reply:
We started doing this after https://github.com/getsentry/sentry-javascript/pull/4519 - to make reading https://github.com/getsentry/sentry-javascript/blob/master/CHANGELOG.md much easier

The discussion was raised with this issue https://github.com/getsentry/sentry-javascript/issues/4493